tests/foreman/ui/test_hostcollection.py

```diff
@@ -52,27 +49,27 @@ def module_lce(module_org):
 
 
 @pytest.fixture(scope='module')
-def module_repos_collection(module_org, module_lce):
-    repos_collection = RepositoryCollection(
+def module_repos_collection(module_org, module_lce, target_sat):
+    repos_collection = target_sat.cli_factory.RepositoryCollection(
         distro=constants.DISTRO_DEFAULT,
         repositories=[
-            SatelliteToolsRepository(),
-            YumRepository(url=settings.repos.yum_1.url),
-            YumRepository(url=settings.repos.yum_6.url),
+            target_sat.cli_factory.SatelliteToolsRepository(),
+            target_sat.cli_factory.YumRepository(url=settings.repos.yum_1.url),
+            target_sat.cli_factory.YumRepository(url=settings.repos.yum_6.url),
         ],
     )
     repos_collection.setup_content(module_org.id, module_lce.id, upload_manifest=True)
     return repos_collection
 
 
-@pytest.fixture(scope='module')
-def module_repos_collection_module_stream(module_org, module_lce):
-    repos_collection = RepositoryCollection(
-        distro=constants.DISTRO_RHEL8,
-        repositories=[YumRepository(url=settings.repos.module_stream_1.url)],
-    )
-    repos_collection.setup_content(module_org.id, module_lce.id, upload_manifest=True)
-    return repos_collection
+# @pytest.fixture(scope='module')
+# def module_repos_collection_module_stream(module_org, module_lce):
+#     repos_collection = RepositoryCollection(
+#         distro=constants.DISTRO_RHEL8,
+#         repositories=[YumRepository(url=settings.repos.module_stream_1.url)],
+#     )
+#     repos_collection.setup_content(module_org.id, module_lce.id, upload_manifest=True)
+#     return repos_collection
```

Comment:
any reason for commenting this over a complete removal?

Reply:
Left for people to review and understand what was there earlier in the fixtures which is now replaced with generic / global fixtures.